src/main/java/racingcar/domain/Name.java

```diff
@@ -0,0 +1,42 @@
+package racingcar.domain;
+
+import java.util.Objects;
+
+public class Name {
+    private static final int MINIMUM_LENGTH_OF_NAME = 1;
+    private static final int MAXIMUM_LENGTH_OF_NAME = 5;
+
+    private final String name;
+
+    public Name(String name) {
+        validateName(name);
+        this.name = name;
+    }
+
+    private void validateName(String name) {
+        int length = name.length();
+
+        if (length < MINIMUM_LENGTH_OF_NAME || length > MAXIMUM_LENGTH_OF_NAME) {
+            throw new IllegalArgumentException(
+                    String.format("자동차의 이름은 %d자 이상, %d자 이하여야 합니다.", MINIMUM_LENGTH_OF_NAME, MAXIMUM_LENGTH_OF_NAME)
+            );
+        }
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
해당 로직에서 괄호를 사용하여 순서를 나타내 준다면 좀 더 가독성 있는 코드가 될 것 같아요!
```suggestion
        if ((o == null) || (getClass() != o.getClass())) return false;
```

Reply:
좋은 의견 감사합니다!! 😀